packages/esm-patient-list-management-app/src/lists-table/lists-table.test.tsx

```diff
@@ -137,16 +124,6 @@ describe('ListsTable', () => {
   it('renders the available patient lists in a datatable', () => {
     const pageSize = 5;
 
-    mockedUsePagination.mockImplementation(
-      () =>
-        ({
-          currentPage: 1,
-          goTo: () => {},
-          results: patientLists.slice(0, pageSize),
-          paginated: true,
-        }) as unknown as PaginationData,
-    );
```

Comment:
The original mistake here was always passing the `patientLists` collection to the mock function. This didn't properly mimic how the underlying function works -  usePagination receives two arguments - an array of data and a `resultsPerPage` value, which is the number of results to render on a page. I've amended the mock to match that contract, and that gets the tests passing again.